packages/web/src/components/edit/fields/TrackMetadataFields.tsx

```diff
@@ -12,6 +12,8 @@ const messages = {
   description: 'Description'
 }
 
+const MAX_DESCRIPTION_LENGTH = 2500
```

Comment:
seems like it would be annoying to change all of these variables. any way to reduce the number of them?